pkg/crc/segment/segment.go

```diff
@@ -168,6 +168,9 @@ func (c *Client) upload(action string, a analytics.Properties) error {
 	if c.config.Get(crcConfig.ConsentTelemetry).AsString() != "yes" {
 		return nil
 	}
+	// Add preset value as property for each event before uploading to segment
+	preset := crcConfig.GetPreset(c.config)
+	a = a.Set("preset", preset)
```

Comment:
Looks like a good fit for being done in `addConfigTraits` instead of `upload`?

Reply:
Or maybe not,`addConfigTraits` is only called to create the identify event, which we try to avoid sending too often. Dunno what's best :)

Reply:
@cfergeau yes `addConfigTraits` we only called once when new user ID is created and not after that, as part of amplitude we need this matrix for every start operation.

Reply:
Well, you could go from operation to user id to preset.

Reply:
@cfergeau what if the user initially use `openshift` preset but then change it to `microshift` I think the config traits are only one time when new userid generated not when a config is changes right? Also I don't have any idea as of now from amplitude side how it can go from operation => uid => preset, right now the matrix we have action events and then filter on those.

Reply:
they want preset for as many actions, so if we can provide this as part of existing events I do not see why 'try to avoid sending too often' matters

Reply:
> I do not see why 'try to avoid sending too often' matters

My question was about putting the code in the right place, rather than adding it to a  method where it looks out of place. And actually, it would fit better in `properties()` rather than `upload`.